src/bundle/linux/deb_bundle.rs

```diff
@@ -109,7 +109,12 @@ fn generate_control_file(
     )?;
     writeln!(&mut file, "Version: {}", settings.version_string())?;
     writeln!(&mut file, "Architecture: {arch}")?;
-    writeln!(&mut file, "Installed-Size: {}", total_dir_size(data_dir)?)?;
+    // deb Installed-Size is size in bytes / 1024
+    writeln!(
+        &mut file,
+        "Installed-Size: {}",
+        total_dir_size(data_dir)? / 1024
```

Comment:
From the doc you linked, looks like this is supposed to be total size / 1024, rounded up?  This is currently rounding down.

One way to do this would be to write `(total_dir_size(data_dir)? + 1023) / 1024` (see https://stackoverflow.com/a/2745086).  Or I guess you could cast to float and use `ceil()`.

Reply:
I'd rather not cast as float, the SO solution looks good